lib/vrl/stdlib/src/values.rs

```diff
@@ -0,0 +1,75 @@
+use ::value::Value;
+use vrl::function::ArgumentList;
+use vrl::function::Compiled;
+use vrl::function::Example;
+use vrl::function::FunctionCompileContext;
+use vrl::prelude::*;
+use vrl::state::TypeState;
+use vrl::Expression;
+use vrl::Function;
+
+fn values(value: Value) -> Resolved {
+    let mut vec: Vec<Value> = Vec::new();
+    let value_btree = value.try_object()?;
+
+    for (_k, v) in value_btree {
+        vec.push(v)
+    }
+    Ok(Value::Array(vec))
```

Comment:
Same trick as above by using the [`BTreeMap::into_values`](https://doc.rust-lang.org/std/collections/struct.BTreeMap.html#method.into_values) and [`Iterator::collect`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect) methods from the standard library:

```suggestion
    let object = value.try_object()?;
    let values = object.into_values();
    Ok(Value::Array(values.collect()))
```